examples/stress_tests/many_glyphs.rs

```diff
@@ -10,18 +13,23 @@ use bevy::{
 };
 
 fn main() {
-    App::new()
-        .add_plugins(DefaultPlugins.set(WindowPlugin {
-            primary_window: Some(Window {
-                present_mode: PresentMode::Immediate,
-                ..default()
-            }),
+    let mut app = App::new();
+    app.add_plugins(DefaultPlugins.set(WindowPlugin {
+        primary_window: Some(Window {
+            present_mode: PresentMode::Immediate,
             ..default()
-        }))
-        .add_plugin(FrameTimeDiagnosticsPlugin::default())
-        .add_plugin(LogDiagnosticsPlugin::default())
-        .add_systems(Startup, setup)
-        .run();
+        }),
+        ..default()
+    }))
+    .add_plugin(FrameTimeDiagnosticsPlugin::default())
+    .add_plugin(LogDiagnosticsPlugin::default())
+    .add_systems(Startup, setup);
+
+    if std::env::args().any(|arg| arg == "recompute-text") {
```

Comment:
Personally I think a flag like `--recompute-text` makes more sense here.

Reply:
Agreed

Reply:
most examples currently just use the name of the configuration, not as a flag:

https://github.com/bevyengine/bevy/blob/main/examples/stress_tests/many_cubes.rs#L54
https://github.com/bevyengine/bevy/blob/main/examples/stress_tests/many_buttons.rs#L38
https://github.com/bevyengine/bevy/blob/main/examples/stress_tests/many_lights.rs#L88

Reply:
Cool cool. Lets stay consistent for now.